src/actionlist/action-list-tree.scss

```diff
@@ -0,0 +1,91 @@
+// tree-view variant
+// renders ActionList with default styling for tree-view
+// collapse/expand icons presents as leadingActions, rotate 90deg
+// connecting vertical lines between collapse groups
+// consistent font-size between nested groups
+
+@mixin treeConnectingLine($left) {
+  position: absolute;
+  left: $left;
+  width: 1px;
+  height: 100%;
+  content: '';
+  background: var(--color-action-list-item-inline-divider);
+}
+
+.ActionList--tree {
+  --ActionList-tree-depth: 1;
+  // TODO: remove with PVC implementation
+  .ActionList-item--subItem > .ActionList-content {
+    font-size: $body-font-size;
+  }
+
+  .ActionList-item {
+    // nesting (infinate levels)
+    // target items inside expanded subgroup
+    &[aria-expanded] {
+      .ActionList--subGroup {
+        position: relative;
+
+        // --ActionList-tree-depth is defined as an inline style referencing the aria-level of each item ie: aria-level="2"
+        // stylelint-disable-next-line selector-max-specificity, max-nesting-depth, selector-max-compound-selectors, primer/spacing
+        .ActionList-content {
+          padding-left: calc(#{$spacer-2} * var(--ActionList-tree-depth));
```

Comment:
```suggestion
          padding-left: calc($spacer-2 * var(--ActionList-tree-depth));
```

Does this work?

Reply:
It compiles correctly, but still getting a lint error. This is definitely a tomorrow problem 😂